### PR TITLE
refactor for lists

### PR DESCRIPTION
**Pull request by mdaxtman**
@rgerstenberger this is a fix for the rendering issue in presentation mode. We'll have to update spectacle so that the theme for list items doesn't have to be put directly onto each list item as I've done here.

### CODE REVIEW

File: src/components/viewer.js [hunk 1]
```diff
@@ -45,15 +48,13 @@ const renderChildren = (nodes) =>
       );
       /* eslint-enable react/prop-types */
     }
-
-    // Wrap ListItems in Appear
-    if (node.type === 'ListItem') {
+    if (node.props.listType) {
+      Tag = (node.props.listType === 'ordered') ? 'ol' : 'ul';
+      console.log(children);
```
Comment (rgerstenberger) [inline src/components/viewer.js]: boo console

Comment (mdaxtman) [inline src/components/viewer.js]: sorry, linter isn't working right on this one.

File: src/components/viewer.js [hunk 2]
```diff
@@ -45,15 +48,13 @@ const renderChildren = (nodes) =>
...
       return (
-        <Appear>
```
Comment (kenwheeler) [inline src/components/viewer.js:52]: No appears anymore?

Comment (mdaxtman) [inline src/components/viewer.js:52]: per Rob, he made a point that the `Appear` will mess with our rendering of PDFs so I removed at his request.

**Comment by rgerstenberger** [comment]
Approved!

**Pull request merged**